common/elasticsearch/src/main/scala/uk/ac/wellcome/elasticsearch/WorksIndexConfig.scala

```diff
@@ -22,17 +26,87 @@ import com.sksamuel.elastic4s.requests.mappings.{
 }
 import com.sksamuel.elastic4s.requests.mappings.dynamictemplate.DynamicMapping
 
+// TODO: Patch back to elastic4s
```

Comment:
Looks like they've changed the way analyzers/filters are done too, so it's a patch + upgrade + change existing code. Great! 🙃

Reply:
And they only upgrade with es version bumps...